app/core/usePrintPDF/usePrintPDF.js

```
@@ -126,12 +128,14 @@ const usePrintPDF = (
       case STATUS.FETCHING_PDF_DATA:
         const fetchPatientOpts = getMainFetchOpts(getTimePrefs(), getPrintOpts(), fetchedUntil);
         dispatch(actions.async.fetchPatientData(api, fetchPatientOpts, patientId));
+        trackMetric('Fetched earlier patient data', { patientID: patientId, count: 1 });
         break;
 
       case STATUS.GENERATING_PDF:
         const queries = getQueries(data, patient, clinicPatient, clinic, getTimePrefs(), getPrintOpts());
```

Biome (2.5.3)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)